src/stylesheets/elements/_buttons.scss

```diff
@@ -22,139 +22,187 @@
   }
 }
 
-// DEPRECATED: Use .sds-button.sds-button--circular
 .sds-button--circle {
   border-radius: 50%;
-  color: color('secondary-darker');
-  border: 0.65px solid darkgray;
-  height: 2rem;
-  width: 2rem;
-  padding: 2px 0 0 1px;
+  color: color("secondary-darker");
+  border: 0;
+  height: units(4);
+  width: units(4);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: unset;
+  padding: initial;
 
   .svg-inline--fa {
     align-self: center;
   }
 
   &.usa-button--hover,
   &:hover {
-    background-color: color('primary-light');
-    color: color('secondary-darker');
+    background-color: color("primary-light");
+    color: color("secondary-darker");
   }
 
   &.usa-button--active,
   &:active {
-    background-color: color('secondary-darker');
-    color: color('white');
+    background-color: color("secondary-darker");
```

Comment:
duplicated below

Reply:
one resolves to `.usa-button:active`  and the other to `sds-button--circle:active` In fact you will find this on every variant

Reply:
sds-button--circle still has the usa-button class in the njk file

Reply:
indeed, as it should